crates/threshold-signature-server/src/backup_provider/tests.rs

```diff
@@ -66,8 +71,13 @@ async fn backup_provider_unit_test() {
     clean_tests();
     initialize_test_logger().await;
 
-    let (_ctx, _api, _rpc, _validator_ips, _validator_ids) =
-        spawn_tss_nodes_and_start_chain(ChainSpecType::IntegrationJumpStarted).await;
+    let TssTestingResult {
+        substrate_context: _ctx,
+        api: _api,
+        rpc: _rpc,
+        validator_ips: _validator_ips,
+        validator_ids: _validator_ids,
+    } = spawn_tss_nodes_and_start_chain(ChainSpecType::IntegrationJumpStarted).await;
```

Comment:
We shouldn't destructure this whole thing if we're not going to use any of the fields.

I'd recommend we do something like: 

```suggestion
let _spawn_result = spawn_tss_nodes_and_start_chain(ChainSpecType::IntegrationJumpStarted).await;
```